spec/factories/needs.rb

```diff
@@ -7,7 +7,7 @@
     contact
     category { Need.categories.values.sample }
     name { Faker::Lorem.sentence }
-    start_on { [nil, Faker::Date.between(from: 2.days.ago, to: 6.days.from_now)].sample }
+    start_on { [nil, Faker::Date.between(from: 1.days.from_now, to: 6.days.from_now)].sample }
```

Comment:
This had a `from` date in the past so that we could see needs that had a start date that had started. Any reason we need to change it?

Reply:
oh yeah I will revert it, no need to change